### PR TITLE
kvrpcpb, tikvpb: add kv_flashback_to_version interface

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

Ref https://github.com/tikv/tikv/issues/13303.

### CODE REVIEW

**Review by HuSharp — APPROVED** [review]

File: proto/raft_cmdpb.proto [hunk 1]
```diff
@@ -99,6 +108,7 @@ message Request {
     DeleteRangeRequest delete_range = 8;
     IngestSSTRequest ingest_sst = 9;
     ReadIndexRequest read_index = 10;
+    FlashbackToVersionRequest flashback_to_version = 11;
```
Comment (Connor1996) [inline proto/raft_cmdpb.proto:111]: why add to raft cmd, it should add to tikv server API. And all the changes are proposed as delete in raft cmd.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by Connor1996** [comment]
@JmPotato merge it until tikv's PR is ready to be merged, otherwise others updating kvproto may encounter compile error.

**Comment by JmPotato** [comment]
> @JmPotato merge it until tikv's PR is ready to be merged, otherwise others updating kvproto may encounter compile error.

Ok!

**Pull request merged**